app/src/main/java/ch/epfl/qedit/view/quiz/QuestionFragment.java

```diff
@@ -74,7 +88,7 @@ private void onQuestionChanged(Quiz quiz, Integer index) {
      * @param question The question that is going to be shown
      * @param index The index of that Question in the Quiz, the question list
      */
-    private void prepareAnswerFormatFragment(Question question, Integer index) {
+    private void prepareAnswerFormatFragment(Question question, Integer index, int correctAnswer) {
```

Comment:
I don't think `correctAnswer` is used in the method, so we should remove it.

Reply:
Yes of course